includes/definitions/coriant.yaml

```diff
@@ -0,0 +1,47 @@
+os: coriant
+text: 'Coriant TNMS'
+type: network
+icon: coriant
+discovery_modules:
```

Comment:
From discovery_module you can remove the following as they are all 0 by default:

loadbalancers
toner
libvirt-vminfo
vmware-vminfo
junose-atm-vp